include/envoy/ssl/context_config.h

```diff
@@ -39,19 +39,37 @@ class ContextConfig {
   virtual const std::string& ecdhCurves() const PURE;
 
   /**
-   * @return The CA certificate file to use for peer validation.
+   * @return The CA certificate to use for peer validation.
    */
-  virtual const std::string& caCertFile() const PURE;
+  virtual const std::string& caCert() const PURE;
 
   /**
-   * @return The certificate chain file used to identify the local side.
+   * @return Path of the CA certificate to use for peer validation or an empty string
```

Comment:
This returns "<inline>" not "" for inline data.  

Sorry, I made my original comment before I'd gotten to the implementation of the interface.

Reply:
Sigh, I should have known better! Thanks.